### PR TITLE
ID-78

### PR DESCRIPTION
**Pull request by xiangzhe0812**
Set PR draft until fix following problem.
1.having trouble with auspost api endpoint, send email to aus post support and waiting for response.
2.need logic how we gone handle response from aus post, (store on DB?)

### CODE REVIEW

File: server-interfaces/src/ausPost.d.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    transaction_id: string;
+  };
+
+  type WatchList = {
+    check_performed: boolean;
+    check_performed_date: string;
+    found: boolean;
+  };
+
+  type FoundSOurces = {
```
Comment (bitcoinbrisbane) [inline server-interfaces/src/ausPost.d.ts:38]:
```suggestion
  type FoundSources = {
```

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
Few typos

File: server-interfaces/src/ausPost.d.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
+declare namespace server {
```
Comment (ahinchliff) [inline server-interfaces/src/ausPost.d.ts:1]: I dont think you need this here. These interfaces describe how the app should interface with the api. Will the app be sending data to Aus Post directly?

File: server/src/services/ausPost/aus-post.d.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+declare namespace ausPostService {
+  interface AusPostService {
+    sendVerification: (data: server.Claims) => Promise<server.SuccessResponse>;
```
Comment (ahinchliff) [inline server/src/services/ausPost/aus-post.d.ts:3]: Oh i see. This is where you should define the types for AU POST

**Review by ahinchliff — CHANGES_REQUESTED** [review]

**Review by ahinchliff — APPROVED** [review]

**Review by rellfy — APPROVED** [review]

**Pull request merged**